src/main/java/org/spongepowered/api/entity/ai/task/AITask.java

```diff
@@ -75,7 +75,7 @@
      * @param other The other task
      * @return True if it can be, false if not
      */
-    boolean canRunConcurrentWith(AITask<O> other);
+    boolean canRunConcurrentWith(AITask<?> other);
```

Comment:
Also note that with generics, at runtime, the only thing you'll know is the lower bound (which is still going to be `AITask<?>` unless you explicitly cast for generics.